client/src/app/ui/modules/head-bar/services/routing-state.service.ts

```diff
@@ -52,8 +52,9 @@ export class RoutingStateService {
      *
      * @param router Angular Router
      */
-    public constructor(router: Router, private location: Location) {
-        router.events
+    public constructor(private router: Router, private location: Location) {
+        this._previousUrl = this._previousUrl ?? this.location.path();
```

Comment:
What's this change for? Seems wrong to me to initialize the previous URL with the current path.

Reply:
Context: The route recognition that also filled in the `previousUrl` was broken so that the event didn't fire. I fixed that but since the event still only fires upon the second navigation, the `previousUrl` for the first navigation would be null (breaking the back link in the edit view if the navigation to the meeting-edit happened directly after a reload)

I agree that this is a bit bad, I'll see if I can change it so that it makes more sense from the outside